src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -441,38 +439,110 @@
+                                        (:tx-token-fees new-path)))]
+    {:from                      (:from-chain new-path)
+     :amount-in-locked          (:amount-in-locked new-path)
+     :amount-in                 (:amount-in new-path)
+     :max-amount-in             "0x0"
+     :gas-fees                  {:gas-price                "0"
+                                 :base-fee                 (convert-to-gwei (:tx-base-fee new-path))
+                                 :max-priority-fee-per-gas (convert-to-gwei (:tx-priority-fee new-path))
+                                 :max-fee-per-gas-low      (convert-to-gwei
+                                                            (get-in
+                                                             new-path
+                                                             [:suggested-levels-for-max-fees-per-gas
+                                                              :low]))
+                                 :max-fee-per-gas-medium   (convert-to-gwei
+                                                            (get-in
+                                                             new-path
+                                                             [:suggested-levels-for-max-fees-per-gas
+                                                              :medium]))
+                                 :max-fee-per-gas-high     (convert-to-gwei
+                                                            (get-in
+                                                             new-path
+                                                             [:suggested-levels-for-max-fees-per-gas
+                                                              :high]))
+                                 :l-1-gas-fee              (convert-to-gwei (:tx-l-1-fee new-path))
+                                 :eip-1559-enabled         true}
+     :bridge-name               (:processor-name new-path)
+     :amount-out                (:amount-out new-path)
+     :approval-contract-address (:approval-contract-address new-path)
+     :approval-required         (:approval-required new-path)
+     :estimated-time            (:estimated-time new-path)
+     :approval-gas-fees         (* (money/wei->ether (get-in new-path
+                                                             [:suggested-levels-for-max-fees-per-gas
+                                                              :medium]))
+                                   (:approval-gas-amount new-path))
+     :to                        (:to-chain new-path)
+     :bonder-fees               bonder-fees
+     :approval-amount-required  (:approval-amount-required new-path)
+     ;;  :cost () ;; tbd not used on desktop
+     :token-fees                token-fees
+     :gas-amount                (:tx-gas-amount new-path)}))
+
+(rf/reg-event-fx :wallet/handle-suggested-routes
+ (fn [_ data]
+   (let [suggested-routes-new-data (cske/transform-keys transforms/->kebab-case-keyword
```

Comment:
I noticed now that `cske/transform-keys` is being used in the wallet context in _production_ code, but there's no usage outside of it. `transform-keys` is slow. It uses `postwalk` and has other overheads we don't need, like caring about metadata. We had at least one performance regression that I remember because of this function, maybe more.

The solution we have been using is to manually transform the keys in `status-im.data-store.*` namespaces. It's less convenient, but it's something I believe we should live with. The recommendation is important enough that it should be in the guidelines.

cc @flexsurfer @ulisesmac @J-Son89

Reply:
is this only with `transform-keys`? How about `cske/->kebab-case-keyword` as we used it in the `utils.transforms` namespace

Reply:
@ilmotta sometimes we need it, for those cases, we are using a memoized version of it (inside `utils.transforms`), it's significantly faster.

Reply:
> even in an unnested map the difference is striking.

Yeah, that's because even if the map is not nested, the Clojure walker pass through the values and not only the keys, at the end if we want to make sure we are properly transforming all the map, it's going to be expensive. I explored this a while ago and I found hard to find a middle point, that's why I only decided to merge the memoized version.